bin/hsd-cli

```diff
@@ -64,6 +64,32 @@ class CLI {
     this.log(info);
   }
 
+  async getNameState() {
+    const name = this.config.str(0, '');
+
+    const ns = await this.client.getNameState(name);
+
+    if (!ns){
+      this.log('NameState not found.');
```

Comment:
Hm, camel case in error messages for the CLI user...? I dunno, the command is all lowercase `namestate`.
Not sure how I feel about it. Although also worth noting, that even a totally random name will return _something_:

```
{
  "start": {
    "reserved": false,
    "week": 31,
    "start": 1152
  },
  "info": null
}
```

So might be moot anyway.